README.markdown

```diff
@@ -202,6 +202,10 @@ Removes the record separator from the end of a string or an array of strings; fo
 
 Returns a new string with the last character removed. If the string ends with '\r\n', both characters are removed. Applying `chop` to an empty string returns an empty string. If you want to merely remove record separators, then you should use the `chomp` function. Requires a string or an array of strings as input. *Type*: rvalue.
 
+#### `clamp`
+
+Keeps value within the range [Min, X, Max] by sort based on integer value. Takes strings, arrays or numerics.  *Type*: rvalue.
```

Comment:
Please explain how strings and arrays are handled. Specifically whether strings are compared numerically or lexically (which locale?) and how arrays figure into it.